### ISSUE TITLE
[BUG] "1" is detected as "&" on azerty keyboard

### ISSUE THREAD

**Issue by piotrpawlik** [JohannesKlauss/react-hotkeys-hook#1026]
**Describe the bug**
"1" shortcut is detected as "&" on azerty keyboard. This might mean it's different on different keyboards. Code example:
```
useHotkeys('1', () => {})
```

**To Reproduce**
1. Click on "1" on azerty keyboard
2. Instead "&" is detected as keyboard sign, not "1"

Link to code sandbox:  https://codesandbox.io/s/determined-waterfall-kco636

**Expected behavior**
"1" hotkey should work across all keyboard layouts 

**Desktop (please complete the following information):**
 - OS: macos 12.4
 - Browser: chrome
 - Version: 112

**Comment by piotrpawlik** [comment]
@JohannesKlauss any idea? :)

**Comment by JohannesKlauss** [comment]
Yes this is because of the confusion of the key and the keyCode. I am working on a version 5 that will only listen to the produced key and falls back to the keyCode if needed.

See the discussion here:
https://github.com/JohannesKlauss/react-hotkeys-hook/discussions/1014

**Comment by piotrpawlik** [comment]
Thanks for reply @JohannesKlauss 👍 It's good to see it being addressed. 
For now I've used `ignoreModifiers: true` which solves the issue for my particular case, but only because I need to use 1-9 keys.

**Comment by JohannesKlauss** [comment]
This is a layout specific problem that gets addressed in version 5.

This has been fixed in pre release [5.0.0-1](https://www.npmjs.com/package/react-hotkeys-hook/v/5.0.0-1). Please try out this version.
To listen to `1` use you just use `useHotkeys('1', callback, {useKey: true})`

Please let me know if that fixes your issue.

**Comment by larafale** [comment]
hello, i installed `"react-hotkeys-hook": "5.0.0-1"`

i'm using `[keys, { start, stop, isRecording }] = useRecordHotkeys({useKey: true})` to capture keyboard shortcuts, but it stills captures QWERTY instead of my french AZERTY keyboard. 

has it been implemented in that hook yet ?

thanks a lot for this lib :) i over use it and love it. hotkeys FTW !

**Comment by JohannesKlauss** [comment]
This is fixed in 5.0.1
